packages/graphql/tests/schema/connect-or-create-id.test.ts

```diff
@@ -108,7 +108,12 @@ describe("connect or create with id", () => {
             }
 
             input ActorMoviesConnectOrCreateFieldInputOnCreate {
-              node: MovieCreateInput!
+              node: ActorMoviesConnectOrCreateFieldInputOnCreateNode!
```

Comment:
Just taking this single example - this feels like we're creating far too many types which are specific to this particular relationship, when this could really just be `MovieOnCreateInput`. I had this roughly working myself when I took a short look at this a while ago, so it should be feasible.

Reply:
That's a good point, I'll take a look at that